tests/unit/utils/test_common.py

```diff
@@ -281,3 +281,9 @@ def forward_request(self, method, path, data, headers):
     # clean up
     proxy.stop()
     rm_rf(tmp_file)
+
+
+def load_test_resource(file_name: str, file_path: str = None) -> str:
```

Comment:
nit: I think this file name is a bit misleading - `test_common.py` is currently used to test the common utils (i.e., not to define utility functions that can be imported from elsewhere). I'd probably move this function into a file `tests/helpers.py`, or into our existing util file `localstack/utils/testutil.py` that already contains some test utilities. 👍

Reply:
I'll keep it in the test suite then, because of the relative paths and until it because an issue maintaining, having it in the class seems to be the pattern to load these files anyway.